cesm/driver/ensemble_driver.F90

```diff
@@ -203,69 +203,68 @@ subroutine SetModelServices(ensemble_driver, rc)
     !-------------------------------------------
 
     allocate(petList(ntasks_per_member))
+    ! which driver instance is this?
+    inst = localPet/ntasks_per_member + 1
```

Comment:
@jedwards4b - very clever! Have you checked if this solved the performance problem?